### ISSUE TITLE
[Help]: Some characteristic values are not shown at 20 bytes approximately every 10ms.

### ISSUE THREAD

**Issue by corteXnzt** [boskokg/flutter_blue_plus#808]
### Requirements

- [X] I've looked at the README 'Common Problems' section

### Have you checked this problem on the example app?

No

### FlutterBluePlus Version

1.31.16

### Flutter Version

3.16.7 

### What OS?

Android

### OS Version

Android 13

### Bluetooth Module

nRF52DK_nrf52832

### What is your problem?

I have set the nrf52dk(peripheral) to minimum connection interval of 10ms.

I have two central device I used for debugging, 1 nrf52 dongle as central and an android 13 Phone.

With the nrf52 dongle the central follows what the peripheral (nrf52dk) connection interval is set which is 10ms. 

With android 13 I have set it to ConnectionPriority.high and connection interval fluctuates from 6-36 ms. 

1. Is their a connection interval setting on flutter blue plus? 

2. And my main issue is the values are not all processed / shown as you can see from the logs attached. onCharacteristicChanged but values are not shown.

### Logs

```shell
D/BluetoothGatt( 5948): onConnectionUpdated() - Device=FF:AF:73:C8:8B:8D interval=6 latency=0 timeout=500 status=0
D/BluetoothGatt( 5948): onConfigureMTU() - Device=FF:AF:73:C8:8B:8D mtu=65 status=0
D/[FBP-Android]( 5948): [FBP] onMtuChanged:
D/[FBP-Android]( 5948): [FBP]   mtu: 65
D/[FBP-Android]( 5948): [FBP]   status: GATT_SUCCESS (0)
I/flutter ( 5948): [FBP] [[ OnMtuChanged ]] result: {error_string: GATT_SUCCESS, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, mtu: 65}
I/flutter ( 5948): [FBP] <discoverServices> args: FF:AF:73:C8:8B:8D
D/[FBP-Android]( 5948): [FBP] onMethodCall: discoverServices
D/BluetoothGatt( 5948): discoverServices() - device: FF:AF:73:C8:8B:8D
I/flutter ( 5948): [FBP] <discoverServices> result: true
D/BluetoothGatt( 5948): onSearchComplete() = Device=FF:AF:73:C8:8B:8D Status=0
D/[FBP-Android]( 5948): [FBP] onServicesDiscovered:
D/[FBP-Android]( 5948): [FBP]   count: 4
D/[FBP-Android]( 5948): [FBP]   status: 0GATT_SUCCESS
I/flutter ( 5948): [FBP] [[ OnDiscoveredServices ]] result: {error_string: GATT_SUCCESS, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, services: [{included_services: [], characteristics: [{descriptors: [{service_uuid: 1801, remote_id: FF:AF:73:C8:8B:8D, descriptor_uuid: 2902, characteristic_uuid: 2a05}], service_uuid: 1801, remote_id: FF:AF:73:C8:8B:8D, characteristic_uuid: 2a05, properties: {broadcast: 0, write_without_response: 0, notify_encryption_required: 0, read: 0, authenticated_signed_writes: 0, extended_properties: 0, indicate: 1, indicate_encryption_required: 0, write: 0, notify: 0}}, {descriptors: [], service_uuid: 1801, remote_id: FF:AF:73:C8:8B:8D, characteristic_uuid: 2b29, properties: {broadcast: 0, write_without_response: 0, notify_encryption_required: 0, read: 1, authenticated_signed_writes: 0, extended_properties: 0, indicate: 0, indicate_encryption_required: 0, write: 1, notify: 0}}, {descriptors: [], service_uuid: 1801, remote_id: FF:AF:73:C8:8B:8D, characteristic_uuid: 2b2a, pr
I/flutter ( 5948): [FBP] <setNotifyValue> args: {remote_id: FF:AF:73:C8:8B:8D, service_uuid: 1801, secondary_service_uuid: null, characteristic_uuid: 2a05, force_indications: false, enable: true}
D/[FBP-Android]( 5948): [FBP] onMethodCall: setNotifyValue
D/BluetoothGatt( 5948): setCharacteristicNotification() - uuid: 00002a05-0000-1000-8000-00805f9b34fb enable: true
I/flutter ( 5948): [FBP] <setNotifyValue> result: true
D/BluetoothGatt( 5948): onConnectionUpdated() - Device=FF:AF:73:C8:8B:8D interval=36 latency=0 timeout=500 status=0
D/[FBP-Android]( 5948): [FBP] onDescriptorWrite:
D/[FBP-Android]( 5948): [FBP]   chr: 2a05
D/[FBP-Android]( 5948): [FBP]   desc: 2902
D/[FBP-Android]( 5948): [FBP]   status: GATT_SUCCESS (0)
I/flutter ( 5948): [FBP] [[ OnDescriptorWritten ]] result: {error_string: GATT_SUCCESS, service_uuid: 1801, success: 1, remote_id: FF:AF:73:C8:8B:8D, descriptor_uuid: 2902, error_code: 0, characteristic_uuid: 2a05, value: 0200}
I/flutter ( 5948): ┌───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
I/flutter ( 5948): │ #0   _SensorViewFPState.discoverRequiredService.<anonymous closure>.<anonymous closure>.<anonymous closure> (package:mobi_pro_force/force_platform/sensor_view_forceplatform.dart:150:22)
I/flutter ( 5948): │ #1   State.setState (package:flutter/src/widgets/framework.dart:1203:30)
I/flutter ( 5948): ├┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄
I/flutter ( 5948): │ 🐛 BLE READY
I/flutter ( 5948): └───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
I/flutter ( 5948): [FBP] <setNotifyValue> args: {remote_id: FF:AF:73:C8:8B:8D, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, secondary_service_uuid: null, characteristic_uuid: 1300, force_indications: false, enable: true}
D/[FBP-Android]( 5948): [FBP] onMethodCall: setNotifyValue
D/BluetoothGatt( 5948): setCharacteristicNotification() - uuid: 00001300-0000-1000-8000-00805f9b34fb enable: true
D/[FBP-Android]( 5948): [FBP] onDescriptorWrite:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP]   desc: 2902
D/[FBP-Android]( 5948): [FBP]   status: GATT_SUCCESS (0)
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
I/flutter ( 5948): [FBP] <setNotifyValue> result: true
I/flutter ( 5948): [FBP] [[ OnDescriptorWritten ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, descriptor_uuid: 2902, error_code: 0, characteristic_uuid: 1300, value: 0100}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP]   chr: 1300
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onScanResult
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnScanResponse ]] result: {advertisements: [{rssi: -99, connectable: 1, appearance: 128, manufacturer_data: {}, remote_id: 5C:62:8B:C5:6A:4E, service_data: {}, service_uuids: [8641]}]}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
I/flutter ( 5948): [FBP] [[ OnCharacteristicReceived ]] result: {error_string: GATT_SUCCESS, service_uuid: 01d4a394-429f-45b4-ae86-6574aa68dd53, success: 1, remote_id: FF:AF:73:C8:8B:8D, error_code: 0, characteristic_uuid: 1300, value: 00000a0014001e00280032003c00460050005a00}
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
D/[FBP-Android]( 5948): [FBP] onCharacteristicChanged:
D/[FBP-Android]( 5948): [FBP]   chr: 1300
```

**Comment by corteXnzt** [comment]
Tested it with the example app and has same scenario which other values are not shown.
![Screenshot from 2024-03-09 14-11-11](https://github.com/boskokg/flutter_blue_plus/assets/49268713/f2e730b0-3e2a-4a2e-bbd3-b055757346bc)

But the sniffer logs shows all values are received. Android also indicates it is received but I wonder why it is not shown. 

Here is the sniffer log in wireshark.  Logs are hard to read when copied and paste so I used screenshots instead.

![Screenshot from 2024-03-09 14-09-02](https://github.com/boskokg/flutter_blue_plus/assets/49268713/9090281e-fa49-4dca-9d72-ac9e59dd5db1)
